### PR TITLE
Ruff toml begone 5

### PR DESCRIPTION
**Pull request by jelly**
These PR's are getting smaller and smaller. Not 100% confident about the `map` changes let's see how it goes, please verify that coverage did not break.

### CODE REVIEW

File: test/verify/check-shell-multi-machine-key [hunk 1]
```diff
@@ -139,7 +139,7 @@ class TestMultiMachineKeyAuth(testlib.MachineCase):
         try:
             m1.execute("ps xa | grep ssh-agent | grep -v grep")
         except subprocess.CalledProcessError:
-            assert False, "No running ssh-agent found"
+            assert False, "No running ssh-agent found"  # noqa: PT015 assertion always fails
```
Comment (martinpitt) [inline test/verify/check-shell-multi-machine-key:144]: That should work with `self.fail("msg")`, no? See https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail

File: test/common/testlib.py [hunk 2]
```diff
@@ -1148,15 +1151,21 @@ def img_eq(ref, now, delta):
                 self.failed_pixel_tests += 1
 
     def assert_pixels(self, selector: str, key: str,
-                      ignore: List[str] = [],
+                      ignore: List[str],
```
Comment (martinpitt) [inline test/common/testlib.py:1155]: I think this needs a `= None`, that may be the cause for many of the failures.

File: test/common/testlib.py [hunk 3]
```diff
@@ -2540,13 +2549,17 @@ def wait(func: Callable, msg: Optional[str] = None, delay: int = 1, tries: int =
...
-def sit(machines={}):
+def sit(machines=None):
     """
     Wait until the user confirms to continue.
 
     The current test case is suspended so that the user can inspect
     the browser.
     """
+    if machines is None:
+        sys.stderr.write("No machines found to sit in")
```
Comment (martinpitt) [inline test/common/testlib.py:2561]: Needs a `\n`. But moreover, I think that'd make more sense as an `assert`?

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Lots of test failures, but I think it's due to the issue below.

File: test/common/testlib.py (hunk 3 above)
Comment (jelly) [reply test/common/testlib.py]: Changed it to an assert, that's a good suggestion.

**Comment by jelly** [comment]
```
  File "/work/bots/make-checkout-workdir/test/common/run-tests", line 489, in run
    create_coverage_report()
  File "/work/bots/make-checkout-workdir/test/common/lcov.py", line 460, in create_coverage_report
    *sum([["--add", f] for f in lcov_files])])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unsupported operand type(s) for +: 'int' and 'list'
```
Coverage cannot be generated.

File: test/verify/check-shell-multi-machine-key [hunk 4]
```diff
@@ -139,7 +139,7 @@ class TestMultiMachineKeyAuth(testlib.MachineCase):
         try:
             m1.execute("ps xa | grep ssh-agent | grep -v grep")
         except subprocess.CalledProcessError:
-            assert False, "No running ssh-agent found"
+            self.fail("No running ssh-agent found")
```
Comment (martinpitt) [inline test/verify/check-shell-multi-machine-key:144]: This is a good fix, just the commit message "add specific noqa comments for PT015" is now wrong. But no need for another round for this.

**Review by martinpitt — APPROVED** [review]
Niice!

**Pull request merged**